o2a/o2a_libs/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "o2a_lib"
```

Comment:
Does this name need to match the name in pypi?

Reply:
@michalmodras yes it is